lte/gateway/python/magma/pipelined/app/enforcement.py

```diff
@@ -496,4 +496,192 @@ def deactivate_rules(self, imsi, ip_addr, rule_ids):
             self._deactivate_flows_for_subscriber(imsi, ip_addr)
         else:
             for rule_id in rule_ids:
-                self._deactivate_flow_for_rule(imsi, ip_addr, rule_id)
\ No newline at end of file
+                self._deactivate_flow_for_rule(imsi, ip_addr, rule_id)
+
+    def _install_default_flow_for_session(self, seid):
```

Comment:
Maybe lets just reuse what we have? 
`_install_default_flow_for_subscriber` does this already

Reply:
Yes. reused it.